### ISSUE TITLE
Suggestion: use Album Artist tag in "Artists" view

### ISSUE THREAD

**Issue by fizzwhiz** [kabouzeid/phonograph-issue-tracker#128]
In the "Artists" view, use names from the Album Artist tag instead of the Artist tag to prevent duplication when a song's artist tag contains more than one name (i.e. main artist plus featured artists). The Album Artist tag usually contains only one name.

**Comment by rocketinventor** [comment]
I disagree, I like to be able to go to each artist and hear all the songs I have from that artist. 
Maybe this could be a setting, instead.

**Comment by rmeissn** [comment]
What's about adding a switch inside settings or quick settings that toggles between "sort by Artist" and "sort by Album Artist". I would also like to sort by Album Artist instead of Artist.
